src/components/DefaultPage/DefaultPage.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from "react";
+import { connect } from "react-redux";
+import { IMovie } from "../../interfaces/IMovie";
+import { loadMovies } from "../../store/actions/loadMovies";
+import { setSearchQuery } from "../../store/actions/searchMovie";
+import { IAppState } from "../../store/reducers/rootReducer";
+import Header from "../Header";
+import Main from "../Main";
+import { RouteComponentProps } from "../MovieDetails/MovieDetails";
+
+interface MatchParams {
+  query: string;
+}
+interface IDefaultPageProps extends RouteComponentProps<MatchParams> {
+  movies: IMovie[];
+  loadMoviesProp: Function;
+  setSearchQueryProp: Function;
+}
+
+const DefaultPage: React.FC<IDefaultPageProps> = (props) => {
```

Comment:
you can destructure props here directly instead of using 'props.something' in the code below. I think it's more clear to do so.